micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/FeatureSearch.js

```
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export const FeatureSearch = ({ className, width = "21", height = "20", style = {}, fill = "#E65006", onClick = null }) => (
+    <svg width={width} height={height} className={className} onClick={onClick} style={style} viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)